### PR TITLE
feat(shown_form): write shown_form data into database, and create ShownForm admin console 

### PR DESCRIPTION
**Pull request by Neilxx**
#68

### CODE REVIEW

File: backend/api/views/factories_cr.py [hunk 1]
```diff
@@ -161,6 +183,9 @@ def _handle_create_ohshown_events(request):
             factory=new_factory, report_record=report_record
         )
         Reporter.objects.create(**new_reporter_field)
+        if post_body.get("type") == '2-1':
```
Comment (tai271828) [inline backend/api/views/factories_cr.py:186]: Hmmm this part is interesting. We may think about how to make it more readable.

**Review by tai271828 — APPROVED** [review]
One open question for your reference (see the inline comment). However, the whole PR is good to go at this development stage.

**Comment by tai271828** [comment]
Thanks @Neilxx for the contribution!!

File: backend/api/views/factories_cr.py (hunk 1 above)
Comment (Neilxx) [reply backend/api/views/factories_cr.py:186]: Use something like enum?
Also type '2-1' or '2-2' kind of redundant.
Integer + enum feels good to me.

**Pull request merged**